module/surveys/helper.js

```diff
@@ -1888,4 +1888,44 @@ module.exports = class SurveysHelper {
         })
     }
 
+
+ /**
+      * Get imported surveys documents.
+      * @method
+      * @name getImportedSurveys
+      * @param  {String} userId - userId of user.
+      * @param  {String} programId - program Id.
+      * @returns {result} - all the survey which user has started in that program. 
+     */
+    static getImportedSurveys(userId, programId){
+        return new Promise(async (resolve, reject) => {
+            try {
+                
+                let surveyData = await this.surveyDocuments({
+                    programId : programId,
+                    createdBy : userId,
```

Comment:
@ankitpws  the condition sequence has to be reversed, createdBy, and then programId, it makes the query faster

Reply:
Done